app/views/overview/_virtual-machine-row.html

```diff
@@ -47,9 +47,9 @@ <h3>
   </div>
   <div class="list-pf-expansion collapse" ng-if="row.expanded" ng-class="{ in: row.expanded }">
     <div class="list-pf-container">
-      <div class="expanded-section resource-details">
-        <h3>Details</h3>
-        <div class="row">
+      <div class="row-expanded-top overview-ovm">
+        <h4 class="component-label section-label">Details</h4>
```

Comment:
"Details" isn't a component, so this is a misuse of this class.  Is it necessary to label the section if there is only the one?  Additionally, the other .component-label.section-label elements are &lt;div&gt;s.  It would be good to follow the existing pattern.  And I think that change may solve your margin issues?